stories/s1grdiw.eocloud.stories.js

```diff
@@ -77,17 +73,13 @@ export const getMapWMS = () => {
   wrapperEl.insertAdjacentElement('beforeend', img);
 
   const perform = async () => {
-    const layer = new S1GRDEOCloudLayer(
+    const layer = new S1GRDEOCloudLayer({
       instanceId,
       layerId,
-      null,
-      null,
-      null,
-      null,
-      AcquisitionMode.IW,
-      Polarization.DV,
-      Resolution.HIGH,
-    );
+      acquisitionMode: AcquisitionMode.IW,
+      polarization: Polarization.DV,
+      resolution: Resolution.HIGH,
```

Comment:
repeated just for easy find: resolution isn't a parameter for S1GRDEOCloudLayer anymore